app/tests/test_user.py

```diff
@@ -0,0 +1,52 @@
+from ..models import User
+from ..serializers import UserSerializer
+from .base import BaseTestCase
+
+
+class UserTestCase(BaseTestCase):
+
+    def test_get_all_users(self):
+        response = self.client.get('/users/')
+        self.assertEqual(response.status_code, 200)
```

Comment:
Maybe you can also assert that you get the users in the structure you are expecting? or that some kind of data exists in it instead of just status code.